Source/SwiftLintFramework/Helpers/Reachability.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+#if !os(Linux)
+import SystemConfiguration
+#endif
+
+public class Reachability {
+    /// Returns whether the device is connected to a network.
+    /// On Linux, this always evaluates to true.
+    public static func isConnectedToNetwork() -> Bool {
+        #if os(Linux)
+        return true
```

Comment:
@jpsim Do you have a better idea how to do this considering `SystemConfiguration` isn't available on Linux...? In the end, it doesn't matter too much, as there's also still the timeout – this just makes it faster to recognize there's no network...

Reply:
Fine to assume always reachable on Linux. 👍